packages/@tinacms/cli/src/cmds/audit/index.ts

```diff
@@ -23,10 +23,19 @@ import { auditCollection, auditDocuments } from './audit'
 import { logger } from '../../logger'
 import chalk from 'chalk'
 import prompts from 'prompts'
+import { Telemetry } from '@tinacms/metrics'
 
 const rootPath = process.cwd()
 
 export const audit = async (ctx: any, next: () => void, options) => {
+  const t = new Telemetry({ disabled: options.noTelemetry })
+  await t.submitRecord({
+    event: {
+      name: 'tinacms:audit:invoke',
```

Comment:
Following the convention from this other ones, maybe this should be `tinacms:cli:audit:invoke`

Reply:
Definitely